### PR TITLE
Update sc-tor.md

### PR DESCRIPTION
**Pull request by mariongalley**
Update to the ToR as approved by the Stewardship Committee

### CODE REVIEW

**Comment by mariongalley** [comment]
I've checked these changes and I'm happy with them. I'm approving this in my capacity as member of 360Giving team, as the first stage in PATCH approval process, which requires sign off from 360Giving team and a member of the Stewardship committee.

File: documentation/about/sc-tor.md [hunk 1]
```diff
@@ -32,33 +37,31 @@
...
    6.2 Any declaration of a conflict of interest is to be noted in the public record of discussions, together with details of any proposals made.
 
-***7\. Individual member’s responsibilities*** 
+***7\. Individual members' responsibilities*** 
 
    7.1 Committee members will at all times:
 
 - Remain appropriately informed on 360Giving’s open data Standard and operating environment.
 - Endeavour to attend all Committee meetings, ensuring that they are adequately prepared to contribute to the discussion.
-- Exercise independence of judgment, acting legally and in good faith to promote and protect 360Giving’s interests.
```
Comment (KDuerden) [inline documentation/about/sc-tor.md:57]: Does this being highlighted red mean it is being deleted? This still appears in the approved version of the ToR.

Comment (mariongalley) [reply documentation/about/sc-tor.md:57]: Thanks for spotting this, I think it's a mistake, you're right it shouldn't be removed

File: documentation/about/sc-tor.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Terms of Reference of the Stewardship Committee
+# Terms of Reference of the 360Giving Data Standard Stewardship Committee
```
Comment (KDuerden) [inline documentation/about/sc-tor.md:2]: Approved by the 360Giving Board of Trustees on 15th July 2021

File: documentation/about/sc-tor.md (hunk 1 above)
Comment (KDuerden) [inline documentation/about/sc-tor.md:57]: - Exercise independence of judgment, acting legally and in good faith to promote and protect 360Giving’s interests.

**Review by KDuerden — CHANGES_REQUESTED** [review]
I have checked this PR which is to update the Terms of Reference on the Standard documentation site to those agreed by the [Stewardship committee on 18 June 2021](https://forum.threesixtygiving.org/t/standard-stewardship-committee-minutes/161/8) and approved by 360Giving Boards on 15th July 2021. 

I request that third bullet in 7.1 (row 57) should not be deleted as this is still part of the text of the approved ToR.

_Exercise independence of judgment, acting legally and in good faith to promote and protect 360Giving’s interests._ 

I have also requested that the approval date be added to the top of the document.

If this feedback actioned, I am happy to approve this PR as a PATCH update, as a backward compatible update to documentation, in my role as member of the Stewardship Committee.

**Review by KDuerden — APPROVED** [review]
The two comments raised have been address so I'm happy to approve this updated PR.

**Pull request merged**